plugins/DynamicsProcessor/DynamicsProcessor.cpp

```diff
@@ -64,8 +64,10 @@ DynProcEffect::DynProcEffect( Model * _parent,
 	m_dpControls( this )
 {
 	m_currentPeak[0] = m_currentPeak[1] = DYN_NOISE_FLOOR;
-	m_rms[0] = new RmsHelper( 64 * Engine::audioEngine()->processingSampleRate() / 44100 );
-	m_rms[1] = new RmsHelper( 64 * Engine::audioEngine()->processingSampleRate() / 44100 );
+	m_rms[0] = new RmsHelper(256 * Engine::audioEngine()->processingSampleRate() / 44100);
```

Comment:
You should not be doing initialization using assignment (refer to [C.49](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c49-prefer-initialization-to-assignment-in-constructors)). These assignments can be done using member-wise initialization.

Also, is there really a need to allocate `RmsHelper` on the heap? I think you could create these on the stack.

Reply:
Issue fixed, I moved m_rms to the stack.